src/components/control/LocationInput/LocationInput.stories.tsx

```diff
@@ -12,11 +12,8 @@ export default meta;
 export const LocationInput: StoryObj = {
   render: () => (
     <>
-      <label htmlFor="custom-location-input">Where are you?</label>
-      <LocationInputComponent
-        input-id="custom-location-input"
-        placeholder="Enter a town or postcode..."
-      ></LocationInputComponent>
+      <label htmlFor="custom-location-input">Enter a town or postcode</label>
+      <LocationInputComponent input-id="custom-location-input"></LocationInputComponent>
```

Comment:
```suggestion
      <LocationInputComponent input-id="custom-location-input" />
```

💅 Can be a bit neater to self close